src/test/compile-fail/empty-struct-with-braces-1.rs

```diff
@@ -8,13 +8,10 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-// compile-flags: -Z parse-only
+// Empty struct defined with braces shouldn't add names into value namespace
```

Comment:
I see the RFC is quite explicit about this design, so that was just something new to me.